assets/model_monitoring/components/src/shared_utilities/span_tree_utils.py

```diff
@@ -226,7 +226,6 @@ def _construct_span_tree(self, spans: List[SpanTreeNode]) -> Optional[SpanTreeNo
                 parent_span = self.get_span_tree_node_by_span_id(parent_id)
                 if parent_span is not None:
                     parent_span.insert_child(span)
-        # TODO: handle logic if root_span has parent_id not null or if we have multiple root_spans
```

Comment:
we now drop all these traces right?

Reply:
Correct. If the root_span is None we don't create a trace log of it.